addons/mail/static/src/messaging/entity/thread_viewer/thread_viewer.js

```diff
@@ -119,6 +138,14 @@ function ThreadViewerFactory({ Entity }) {
         // Private
         //----------------------------------------------------------------------
 
+        /**
+         * @private
+         */
+        _stopLoaderTimeout() {
```

Comment:
IMO this method should set `isShowingLoading` to `false` and remove the timeout (also rename function appropriately)

Reply:
`_hideLoading` ?

Reply:
Don't like "hide"...
Something like `_prepareLoading()`/`_stopLoading()`: former is a delayed "set loading", while second is an "unset loading".
Each of them handle `isShowingLoading` and timeout accordingly.